src/commands/data/query.ts

```diff
@@ -57,23 +55,10 @@ export class DataSoqlQueryCommand extends SfCommand<unknown> {
       aliases: ['usetoolingapi'],
       deprecateAliases: true,
     }),
-    bulk: Flags.boolean({
```

Comment:
Removing any of these flags will be a breaking change without going through the standard deprecation process... even then, this decision will be made by our PM.

These flags will be required before merging

Reply:
ok, when your PM decides to take over this hidden bug, you can contact me again and I will do the changes